travharv/store.py

```diff
@@ -93,13 +42,13 @@ def select_subjects(self, sparql) -> List[str]:
         log.debug(f"length list_of_subjects: {len(list_of_subjects)}")
         return list_of_subjects
 
-    def verify_path(self, subject, property_path, prefixes=None):
-        sparql = self._qryBuilder.build_syntax(
+    def verify_path(self, subject, property_path, NSM):
+        pre_sparql = self._qryBuilder.build_syntax(
```

Comment:
pre_sparql is an odd name

also because the odd coincidence with the pre in prefix... it could sound like the opposite of what it is: so if one in the wild would have to guess which of pre_sparql or sparql is the one that actually holds the pre_fixes?

but mostly -- imho a perfect / acceptable case to just reuse the sparql var before and after the resolving